import/marc_local.properties

```diff
@@ -72,7 +72,7 @@ author_synonyms = 101a
 publisher = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishers
 publisher_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.IxTheoPublisher), getPublishersOrUnassigned
 
-topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x)
+topic_facet = custom(de.unituebingen.ub.ubtools.solrmarcMixin.TuelibMixin), getValuesOrUnassigned(600x:610x:611x:630x:648x:650a:650x:651x:655x:689abctnp)
```

Comment:
In der aktuellen Implementierung werden bereits die Unterfelder zu einem String zusammengefasst. Wenn man sich die ersten Treffer für ein alleinstehendes "Bibel" in der Facette im Internformat ansieht, zeigt sich, dass hier tatsächlich derzeit kein Buch oder Vers angegeben zu sein scheint.